modules/tensor_mechanics/src/materials/ComputeIsotropicElasticityTensor.C

```diff
@@ -34,6 +34,26 @@ ComputeIsotropicElasticityTensor::ComputeIsotropicElasticityTensor(
     _shear_modulus(_shear_modulus_set ? getParam<Real>("shear_modulus") : -1),
     _youngs_modulus(_youngs_modulus_set ? getParam<Real>("youngs_modulus") : -1)
 {
+  int num_elastic_constants = _bulk_modulus_set + _lambda_set + _poissons_ratio_set +
```

Comment:
should be `unsigned int` unless you expect a negative number of elastic constants...